src/main/webapp/resources/init_db.sql

```diff
@@ -15,6 +15,7 @@ CREATE TABLE `users`
     `login`    VARCHAR(225)   NOT NULL,
     `password` VARCHAR(225)   NOT NULL,
     `deleted`  TINYINT        NOT NULL DEFAULT true,
+    `salt`     VARBINARY(500) NOT NULL,
```

Comment:
Why the size is 500? We definitely know that we will have an array of 16 elements. Or I misunderstood that?

Reply:
we have array of 16 bytes. so it will be more than 16 symbols

Reply:
this number is the size in bytes so you can write like this VARBINARY(16)